### ISSUE TITLE
Small Items in Overworld Not Shuffling (4.2.4/4.2.6)

### ISSUE THREAD

**Issue by MiR4i** [Ellendar/Z2Randomizer#103]
Enabling "Shuffle Small Items" does not shuffle around the small items on the overworld tiles.

Any flagset/seed should show the issue, but in addition to everything I've been playing recently, here are three I specifically checked tonight:

Flagset: `RAAN6AAFesXtN1Tdt0Y$8v4cYf4XT7AALh`
Seeds:
```
537781406
770625760
485798627
```

While I'm sure these are listed in the code somewhere, these are the tiles as they're labeled in the editor:
```
west_hyrule 43 - PBag
west_hyrule 46 - PBag
west_hyrule 51 - 1-Up + Red Jar
west_hyrule 56 - Red Jar
west_hyrule 57 - Red Jars

east_hyrule 33 - PBag
east_hyrule 38 - PBag
east_hyrule 44 - PBag
east_hyrule 51 - 1-Up
east_hyrule 57 - 1-Up
east_hyrule 58 - Red Jar
east_hyrule 62 - PBag + Red Jar
```

**Comment by Ellendar** [comment]
APPARENTLY I disabled this at some point for some reason, probably because it was breaking something back in development and I forgot I even did it and never turned it back on. There are procedures in place today to prevent this from happening, but this is so long ago I don't remember even doing it. Likely pre 4.1 release. Anyway it turns out of you don't explicitly say not to do it, it works...

**Comment by MiR4i** [comment]
It looks as if this the fix for this was lost in the code base debacle that occurred with the release of 4.2.6, and small items in the overworld are, again, not shuffling between locations.

**Comment by Ellendar** [comment]
Fixed for 4.2.7 (no github)
